data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+	on offer
+		event "mooncreek main outfitters" 15
+		conversation
+			`On your arrival to Mooncreek with the Hai convoy, you are surprised to see a slightly put-out looking Alondo waiting for you at the docks.`
+			`	"<first>! How's it going?"`
+			choice
+				`	"Fine, I guess?"`
+				`	"It's going well."`
+			`	"Good, good," he says, "now I know that the last thing you probably want to do right now is traipse from one end of space to the other, but I need to ask you a favour."`
+			`	He finds somewhere to sit down with you and begins, "So you may recall I was helping the Free Worlds set up their diplomatic corps. Obviously I wanted no part of the ambassadorial business to the Republic, but when all this Hai business started up well, let's just say they struggled to find anyone qualified. As it turns out someone in the senate doesn't particularly like me and they've been using my extended time away up here to try and influence the diplomatic corps. Meanwhile, the Hai want the Free Worlds to have some involvement with this station we're on so that everyone is tied in, but obviously the Free Worlds don't particularly care when they could hardly be further from it."`
+			`	"I have devised a compromise, where the Free Worlds will not supply ships to the garrison here but they will instead sponsor both Kraz and Delta V to set up outfitters here, with full upfront costs paid by the Free World government. A defining success will allow me to return to the corps with the influence I need to counter this senator's influence. Otherwise I fear I shall be stuck here on the other side of space for the rest of my career."`
+			`	You look at your friend and you can tell he's waiting for you to accept his premise first, holding something back...`
+			choice
+				`	"What's the catch?"`
+			`	He chuckles, "You're getting to know me too well."`
+			`	"The catch is this, the Free Worlds will only sponsor the outfitters if you agree to escort their convoys directly from Solace and Rust the entire way here. They want to make a big show of it, or rather, a certain senator has goaded them into making a show of it.'`
+			`	He sighs, "On the plus side though, if you're interested, you do get paid diplomatic escort rates."`
+			choice
+				`	"I'll do it friend."`
+					accept
+				`	"Of course I'll do it, but how much is that exactly?"`
+			`	He winks, "A hundred thousand credits per day, for a fifty-two day route with eight stops."`
+			`	"Of course, if you really don't want to do it I'll understand. I don't imagine you're keen to deal with the politics."`
+			`	You have a think about helping a friend... for six million credits.`
+			choice
+				`	"Don't be silly, of course I'll do it friend."`
+					accept
+				`	"I'm sorry Alondo, maybe I can find some other way of helping you later."`
+					decline
+			
+		
+
+mission "HRF 11 Escort Free World Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
```

Comment:
Especially could use a different description, since Mooncreek won't even be on marked on the map during this mission.